src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -258,6 +258,58 @@ public void should_warn_a_task_with_multiple_categories() {
         taskToTearDown = taskId;
     }
 
+    /**
+     *  This FT sends additionalData to DMN in json format to evaluate appealType.
+     *  Disabled as the checkFeeStatus task is created with delayUntil to 28 days and can't be
+     *  retrieved until the task is active.
+     *  When the DMN is deployed onto new wa jurisdiction, this test can be enabled
+     *  with delayUntil as 0.
+     */
+    @Test
+    @Ignore("CheckFeeStatus task cannot be retrieved")
+    public void given_event_submitAppeal_when_appealType_sent_as_json_then_initiate_task() {
+        String caseId = UUID.randomUUID().toString();
+
+        sendMessageWithAdditionalData(
+            caseId,
+            "submitAppeal",
+            "",
+            "appealSubmitted",
+            false
+        );
+
+        waitSeconds(10);
```

Comment:
I'd suggest not doing an explicit wait and using a polling mechanism instead

Reply:
wait time not required.